ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -0,0 +1,225 @@
+}
+
+///Properties that make up list content
+@objc(MSFListVnextState)
+public class MSFListVnextState: NSObject, ObservableObject {
+    @objc @Published public var sectionTitle: String?
+    @objc @Published public var hasBorder: Bool = false
+}
+
+@objc(MSFListIconVnextStyle)
+/// Pre-defined styles of icons
+public enum MSFListIconVnextStyle: Int, CaseIterable {
+    case none
+    case iconOnly
+}
+
+@objc(MSFListCellVnextHeight)
+/// Pre-defined layout heights of cells
+public enum MSFListCellVnextLayoutType: Int, CaseIterable {
+    case oneLine
+    case twoLines
+    case threeLines
+
+    var height: CGFloat {
+        switch self {
+        case .oneLine:
+            return 48
+        case .twoLines:
+            return 64
+        case .threeLines:
+            return 84
+        }
+    }
+}
+
+public class MSFListTokens: ObservableObject {
+    @Published public var backgroundColor: UIColor!
+    @Published public var borderColor: UIColor!
+    @Published public var disclosureIconForegroundColor: UIColor!
+    @Published public var iconColor: UIColor!
+    @Published public var leadingTextColor: UIColor!
+    @Published public var subtitleColor: UIColor!
+    @Published public var trailingItemForegroundColor: UIColor!
+
+    @Published public var highlightedBackgroundColor: UIColor!
+
+    @Published public var borderSize: CGFloat!
+    @Published public var disclosureInterspace: CGFloat!
+    @Published public var disclosureSize: CGFloat!
+    @Published public var horizontalCellPadding: CGFloat!
+    @Published public var iconInterspace: CGFloat!
+    @Published public var iconSize: CGFloat!
+    @Published public var largeIconSize: CGFloat!
+    @Published public var subtitleFont: UIFont!
+    @Published public var textFont: UIFont!
+
+    var iconStyle: MSFListIconVnextStyle!
+    var layoutType: MSFListCellVnextLayoutType!
+
+    public init(layoutType: MSFListCellVnextLayoutType, iconStyle: MSFListIconVnextStyle) {
+        self.layoutType = layoutType
+        self.iconStyle = iconStyle
+        self.themeAware = true
+
+        didChangeAppearanceProxy()
+    }
+
+    @objc open func didChangeAppearanceProxy() {
+        let appearanceProxy: ApperanceProxyType
+
+        if iconStyle == MSFListIconVnextStyle.iconOnly {
+            appearanceProxy = StylesheetManager.S.IconOnlyListTokens
+        } else {
+            appearanceProxy = StylesheetManager.S.MSFListTokens
+        }
+
+        backgroundColor = appearanceProxy.backgroundColor.rest
+        borderColor = appearanceProxy.borderColor
+        disclosureIconForegroundColor = appearanceProxy.disclosureIconForegroundColor
+        iconColor = appearanceProxy.iconColor
+        leadingTextColor = appearanceProxy.labelColor
+        subtitleColor = appearanceProxy.sublabelColor
+        trailingItemForegroundColor = appearanceProxy.trailingItemForegroundColor
+
+        highlightedBackgroundColor = appearanceProxy.backgroundColor.pressed
```

Comment:
I think the press effect (using the highlightedBackgroundColor while the cell is being pressed) is one of the important things we need to have for our MVP implementation.